<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPassWord</class>
 <widget class="QDialog" name="ForgotPassWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_FP_EMail"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_FP_UserName"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_forgepassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>forget password</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_DisplayPassWord">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;ENTER YOUR USERNAME AND EMAIL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
